drivers/i2c/i2c_mcux_lpi2c.c

```diff
@@ -240,17 +261,17 @@ static const struct i2c_driver_api mcux_lpi2c_driver_api = {
 	static struct mcux_lpi2c_data mcux_lpi2c_data_##n;		\
 									\
 	DEVICE_DT_INST_DEFINE(n, &mcux_lpi2c_init, device_pm_control_nop,\
-			    &mcux_lpi2c_data_##n,			\
-			    &mcux_lpi2c_config_##n, POST_KERNEL,	\
-			    CONFIG_KERNEL_INIT_PRIORITY_DEVICE,		\
-			    &mcux_lpi2c_driver_api);			\
+				&mcux_lpi2c_data_##n,			\
```

Comment:
In general please try to avoid unrelated whitespace changes in commits.  No need to back this one out.

Reply:
Removed the whitespace changes.